scene/gui/scroll_container.cpp

```diff
@@ -576,20 +625,27 @@ void ScrollContainer::_bind_methods() {
 	ADD_SIGNAL(MethodInfo("scroll_ended"));
 
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "follow_focus"), "set_follow_focus", "is_following_focus");
+	ADD_PROPERTY(PropertyInfo(Variant::VECTOR2, "follow_focus_buffer"), "set_follow_focus_buffer", "get_follow_focus_buffer");
 
 	ADD_GROUP("Scroll", "scroll_");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "scroll_horizontal"), "set_h_scroll", "get_h_scroll");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "scroll_vertical"), "set_v_scroll", "get_v_scroll");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "horizontal_scroll_mode", PROPERTY_HINT_ENUM, "Disabled,Auto,Always Show,Never Show"), "set_horizontal_scroll_mode", "get_horizontal_scroll_mode");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "vertical_scroll_mode", PROPERTY_HINT_ENUM, "Disabled,Auto,Always Show,Never Show"), "set_vertical_scroll_mode", "get_vertical_scroll_mode");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "scroll_deadzone"), "set_deadzone", "get_deadzone");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "scroll_wheel_sensitivity"), "set_scroll_wheel_sensitivity", "get_scroll_wheel_sensitivity");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "pan_gesture_sensitivity"), "set_pan_gesture_sensitivity", "get_pan_gesture_sensitivity");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "scroll_damping"), "set_scroll_damping", "get_scroll_damping");
```

Comment:
These and setter functions probably should have `PROPERTY_HINT_RANGE` and checks to prevent setting non-positive values.